.github/workflows/release.yml

```diff
@@ -104,6 +107,15 @@ jobs:
           workflow: ci.yml
           workflow_conclusion: success
 
+      - name: ${{ matrix.component.Name }} -- Upload assets
+        uses: softprops/action-gh-release@v1
+        if: startsWith(github.ref, 'refs/tags/')
+        with:
+          files: |
+            ./AppManifest.json
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
       - name: ${{ matrix.component.Name }} -- Upload AppManifest.json to artifacts"
```

Comment:
Please remove uploading to the Artefacts, this step is not necessary anymore. That would be replaced by uploading to the Assets.

Reply:
Done